### PR TITLE
Welcome to Prestige

### PR DESCRIPTION
**Pull request by DIzosimov**
[PT-Link](https://www.pivotaltracker.com/story/show/168988777)

# PR Content
- Welcome page feature test
- Basic step definitions
- Assertion step definitions
- Adds NavBar
- Adds home button and login button
- Styling to home page
- Imports tailwindcss

## What I have Learned
- How to use tailwind
- How to get past unclear errors while styling

---

## Screenshots
Screenshot from 2019-10-09 12-48-58

CSS :)

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by miyeaier — APPROVED** [review]

File: app/views/layouts/_nav.html.haml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    = image_tag "http://fusioncoedsoccer.com/media/bearleague/bl1477363156652.png", :width => "20%"
+    %a.span.font-semibold.text-xl.tracking-tight{:href => root_path} Prestige Worldwide
+  .block.lg:hidden
+    %button.flex.items-center.px-3.py-2.border.rounded.text-teal-200.border-teal-400.hover:text-white.hover:border-white
+      %svg.fill-current.h-3.w-3{:viewbox => "0 0 20 20", :xmlns => "http://www.w3.org/2000/svg"}
+        %title Menu
+        %path{:d => "M0 3h20v2H0V3zm0 6h20v2H0V9zm0 6h20v2H0v-2z"}
+  .w-full.block.flex-grow.lg:flex.lg:items-center.lg:w-auto
+    .text-sm.lg:flex-grow
+      -# %a.block.mt-4.lg:inline-block.lg:mt-0.text-teal-200.hover:text-white.mr-4{:href => root_path}
```
Comment (faraznaeem) [inline app/views/layouts/_nav.html.haml]: Please clean out the comments

File: app/views/layouts/application.html.haml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+!!!
+%html
+  %head
+    %meta{content: 'text/html; charset=UTF-8', http: {equiv: "Content-Type"}}/
+    %title PrestigeWorldwide
+    = csrf_meta_tags
+
+    = stylesheet_link_tag 'application', media: 'all', data: {'turbolinks-track' => 'reload'}
+    = javascript_pack_tag 'application', data: {'turbolinks-track' => 'reload'}
+    = stylesheet_pack_tag 'application'
```
Comment (faraznaeem) [inline app/views/layouts/application.html.haml:10]: Please use new ruby syntax and not hash rockets `=>`

File: app/views/welcome/index.html.haml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+%h1#titleh1 Prestige Worldwide
+
+.max-w-sm.rounded.overflow-hidden.shadow-lg#card
+  %img.w-full{:alt => "Sunset in the mountains", :src => "http://fusioncoedsoccer.com/media/bearleague/bl1477363156652.png"}/
```
Comment (faraznaeem) [inline app/views/welcome/index.html.haml]: Please use new ruby syntax and not hash rockets `=>`

**Review by shsteindorsson — APPROVED** [review]

**Review by faraznaeem — CHANGES_REQUESTED** [review]
Preferrably in the future you can add the styling gem as a separate chore and PR, as that is needed throughout the application

File: app/views/layouts/_nav.html.haml (hunk 1 above)
Comment (DIzosimov) [reply app/views/layouts/_nav.html.haml]: Will do :)

File: app/views/layouts/application.html.haml (hunk 2 above)
Comment (DIzosimov) [reply app/views/layouts/application.html.haml:10]: Donezo :)

File: app/views/welcome/index.html.haml (hunk 3 above)
Comment (DIzosimov) [reply app/views/welcome/index.html.haml]: Donezo :)

**Review by shsteindorsson — APPROVED** [review]

**Review by faraznaeem — APPROVED** [review]

File: features/welcome_page.feature [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Feature: Visit welcome page
+  As a user
+  When I visit the landing page
```
Comment (faraznaeem) [inline features/welcome_page.feature]: this user story is totally off
```
As a user
In order to learn about the offerings 
I would like to learn more about the app 
```

**Review by faraznaeem — CHANGES_REQUESTED** [review]

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**